databricks-geococoding/README.md

```diff
@@ -0,0 +1,41 @@
+
+## Data
+This sample does not include the data, but does show how to integrate with the Precisely Data Experience.  Using your credentials you will be assured to be using the most recent data vintage.  Also this expidites the setting up of data on your cluster.
+
+* **Geocoding Reference Data**: USA-MasterLocationData, USA-TomTom. 
+
+## Setting up DataExperience integration
+  1) Collect your Data Experience API Key and Secret Key by visiting https://data.precisely.com/autodownload  You will need these for the Geocoding Installation Workspace.
+  2) The DataExperience SDK has been prebuilt and included in the repo at /lib/precisely-bigdata-pdx-sdk2.7-full.jar.  It will need to be added to your Databricks FileStore https://docs.databricks.com/data/filestore.html.  You will need to set the location of the file in Configuration and Variables cell of Geocode Installation workbook.
+
+## Building the sample
+The sample is broken into 2 workspaces.  One for installing the other for configuring and executing Geocodes.
+
+## Installing Geocoding
+The installation is split up between multiple cells in the Geocoding Installation Workspace.  For best results you should execute each cell individually in top down order.  
+Most of the cells can be executed as written, only the first has mandatory edits needed.
+
+Configuration and Variables cell builds a shell script to hold values needed by the other cells. This will be the main cell that will require your edits.
+Set SDM_CLASSPATH to the FileStore location of the precisely-bigdata-pdx-sdk2.7-full.jar.
+Set PB_API_KEY to the DataExperience API key from above
+Set PB_SECRET to the DataExperience Secret key from above
+
+Set SDK_URL to any valid/accessible URL location of the spectrum-bigdata-geocoding-VERSION.zip. 
+You will also need to provide an URL for the location of the spectrum-bigdata-geocoding-VERSION.zip. In our example we have a mocked up presigned URL to an s3 repository.  See these links for information on creating aws presigned https://docs.aws.amazon.com/cli/latest/reference/s3/presign.html and for azure https://docs.microsoft.com/en-us/rest/api/storageservices/Service-SAS-Examples?redirectedfrom=MSDN.  If you prefer add the spectrum-bigdata-geocoding-VERSION.zip. to your dbfs and use a file:/// URL.
+
+Once those changes have been made you can execute the cell.
+
+After executing Create Geocoding Library & Attach to Cluster cell you will see two jars called out in the results.  You must pick the appropriate version depending on the spark version you are using.  That jar should be added as a library to your cluster https://docs.databricks.com/libraries/index.html.
+
+The rest of the cells can be executed in order with no other interaction required.
+
+
+## Executing the Demo
+After all cells of the Installation have completed successfully you can move on to Executing the cells in the Demo workbook.
+For this notebook you may need no alterations.  
```

Comment:
is it worth mentioning that they have to change just the base path if they modified it in the installation?  At least mention it follows a similar format - configuration command followed the "actual work"

Reply:
changed